frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoService.java

```diff
@@ -0,0 +1,90 @@
+package ch.shiftcrypto.bitboxapp;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.Service;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.Build;
+import android.os.IBinder;
+
+import androidx.core.app.NotificationCompat;
+
+import goserver.GoAPIInterface;
+import goserver.GoEnvironmentInterface;
+import goserver.Goserver;
+
+public class GoService extends Service {
+    // Binder given to clients
+    private final IBinder binder = new GoServiceBinder();
+
+    private Boolean started = false;
+
+    private final String channelId = "21";
+
+    private final int notificationId = 8;
+
+    @Override
+    public void onCreate() {
+        Util.log("GoService onCreate()");
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationChannel channel = new NotificationChannel(
+                    channelId,
+                    "BitBoxApp",
+                    NotificationManager.IMPORTANCE_LOW
+            );
+            channel.setDescription("BitBoxApp service notification channel");
+            NotificationManager manager = getSystemService(NotificationManager.class);
+            manager.createNotificationChannel(channel);
+        }
+
+        Notification notification = new NotificationCompat.Builder(this, channelId)
+                .setContentTitle("BitBoxApp Go backend service")
+                .setContentText("Running in the background")
```

Comment:
should it say 'running in the foreground' (says foreground below)?

Reply:
I actually think that from the user perspective (title and text are shown in the notification) it is still something going on in the background, even if technically is a foreground service. I slightly changed the title, trying to make it simpler to understand for the user. Feel free to suggest anything different, though :)